styles/Canonical/003-Ubuntu-names-versions.yml

```diff
@@ -0,0 +1,29 @@
+extends: existence
+message: |
+  Please use the Ubuntu product name correctly. E.g. Ubuntu 22.04 LTS, Ubuntu
+  22.10, Ubuntu 23.04 (Lunar Lobster), Ubuntu 23.10 (Mantic), Ubuntu 24.04 LTS
+  (Noble Numbat).
+link: https://docs.ubuntu.com/styleguide/en/#ubuntu
+nonword: true
+ignorecase: true
+scope: raw
+level: suggestion
+tokens:
+  # LTS releases should be followed by "LTS" suffix.
+  - 'Ubuntu (24|22|20|18)\.04(?! LTS)'
+  # Non-LTS releases should not be followed by "LTS" suffix.
+  - 'Ubuntu (23\.10|23\.04|22\.10|21\.10|21\.04|20\.10|19\.10|19\.04|18\.10|17\.10|17\.04) LTS'
+  # All releases should match the code name.
```

Comment:
This regex checks for incorrect usages of Ubuntu. If the Ubuntu version is matched and a parentheses follows, the check will fail if anything other than the code name is not found in the parentheses.